### DISCUSSION TITLE
Custom Fields + Licenses

### DISCUSSION THREAD

**Discussion by northcide** [snipe/snipe-it#8907] [General]
I just started populating my customers data and I'm not able to find a way to associate custom field(sets) with Licenses. Is this a limitation?  This customer in particular has tons of different software, many of which have several important pieces of data to account for them such as serial, subscription ID, contract number, etc.  

I was expecting that I'd be able to just associate a field set. Was I mistaken?
Thanks

**Comment by snipe** [comment]
Hi @northcide - sorry for the delay - we're still figuring out how this new Discussions thingie works.

We unfortunately don't support custom fields in licenses right now. It's on the short-term roadmap though, for sure. We have some refactoring to do to some of the models and then we'll be ready to extend out custom fields to things other than assets.

**Reply by liamf91** [reply]
@snipe Has this been put on the back burner or will it be coming in the upcoming V7? I can still see that custom fields only apply to assets and not license. I am hoping to use them for ISO reports.

**Comment by mauro-miatello** [comment]
This is an old discussion that I think can be closed.
However I want to add my opinion, because I have the same need :)
In my case, I still trying to understand how to better solve this request
I created an asset "Software", and added some custom fields to it
The license is like a property of this asset software, that is assigned to the assets where it is installed.

So, instead of having license assigned to an hardware, 
my idea is to have a license assigned to a software that is assigned to an hardware.
I think that we need like a "best practices" manual.
